### DISCUSSION TITLE
Is there a character limit?

### DISCUSSION THREAD

**Discussion by ExoFi-Labs** [jas3333/GPT-Helper#14] [Q&A]
I have been working with the chatbot on writing some HTML and CSS Code which has been great so far but I think it's starting to hit a word limit around 140 Words? Is there a word / Character limit and if so if there any where to adjust it? I saw somewhere that its possible to increase it with ChatGPT so not sure if it's possible here too? 

Amazing little app btw! :)

**Comment by jas3333** [comment]
I believe there is a 4k token limit per request. It's something I need to look more into. There is a slider  to increase it, which will allow GPT to increase the response size(I think), though I have only ever gotten it to respond with 2k tokens once.  There is quite a bit to still learn about how to use the API. I've just been busy trying to learn authentication so haven't had much time to mess with this more.

And thanks, it was a lot of fun building it. I learned quite a bit. I also made one with a backend that saves conversations if you're familiar with how to setup mongdb you could give it a try. [Repo](https://github.com/jas3333/GPT-Chat)
